### DISCUSSION TITLE
Website - Direct visitors to specific lists

### DISCUSSION THREAD

**Discussion by avoidthehack** [blocklistproject/Lists#898] [Ideas]
Hi.

This is a suggestion for the blocklist.site website (https://blocklist.site).

All of the blocking categories under the "Lists" drop down point to https://github.com/blocklistproject/Lists where they should point to the raw text files instead, like so:

- Advertising -> https://raw.githubusercontent.com/blocklistproject/Lists/master/ads.txt
- Phishing -> https://raw.githubusercontent.com/blocklistproject/Lists/master/phishing.txt
- Tracking -> https://raw.githubusercontent.com/blocklistproject/Lists/master/tracking.txt
- Fraud -> https://raw.githubusercontent.com/blocklistproject/Lists/master/fraud.txt
- Porn -> https://raw.githubusercontent.com/blocklistproject/Lists/master/porn.txt
- Gambling -> https://raw.githubusercontent.com/blocklistproject/Lists/master/gambling.txt
- Scams -> https://raw.githubusercontent.com/blocklistproject/Lists/master/scam.txt
- Malware -> https://raw.githubusercontent.com/blocklistproject/Lists/master/malware.txt
- TikTok -> https://raw.githubusercontent.com/blocklistproject/Lists/master/tiktok.txt

I think this will help visitors who are quickly looking for specific list categories versus getting directed to the general GitHub page where they have to "hunt" for the lists themselves; there are already several links pointing to the general GitHub page ("Explore All" and the GitHub link) for users who want to go there.

**Comment by gap579137** [comment]
> Hi.
> 
> This is a suggestion for the blocklist.site website (https://blocklist.site).
> 
> All of the blocking categories under the "Lists" drop down point to https://github.com/blocklistproject/Lists where they should point to the raw text files instead, like so:
> 
> * Advertising -> https://raw.githubusercontent.com/blocklistproject/Lists/master/ads.txt
> * Phishing -> https://raw.githubusercontent.com/blocklistproject/Lists/master/phishing.txt
> * Tracking -> https://raw.githubusercontent.com/blocklistproject/Lists/master/tracking.txt
> * Fraud -> https://raw.githubusercontent.com/blocklistproject/Lists/master/fraud.txt
> * Porn -> https://raw.githubusercontent.com/blocklistproject/Lists/master/porn.txt
> * Gambling -> https://raw.githubusercontent.com/blocklistproject/Lists/master/gambling.txt
> * Scams -> https://raw.githubusercontent.com/blocklistproject/Lists/master/scam.txt
> * Malware -> https://raw.githubusercontent.com/blocklistproject/Lists/master/malware.txt
> * TikTok -> https://raw.githubusercontent.com/blocklistproject/Lists/master/tiktok.txt
> 
> I think this will help visitors who are quickly looking for specific list categories versus getting directed to the general GitHub page where they have to "hunt" for the lists themselves; there are already several links pointing to the general GitHub page ("Explore All" and the GitHub link) for users who want to go there.

This is actually something we want to do and are currently working on getting done. I am hoping that it will be in the next push.

If you have any more suggestions we are all ears!

**Reply by avoidthehack** [reply]
Only here to help. :)

Also FYI:

- There's a typo on the homepage (comeplete -> complete)
- Typo in the Lists drop down  (Explorer All -> Explore All)

Since there are a lot of different lists, you may also want to consider grouping them in broader categories (like Facebook, TikTok, Twitter all under "Social Media")

I'll be submitting a pull request to fix some minor things I noticed with the github readme - is it appropriate to open an issue and link the pull request? Or just submit the pull?

**Reply by gap579137** [reply]
> Only here to help. :)
> 
> Also FYI:
> 
> * There's a typo on the homepage (comeplete -> complete)
> * Typo in the Lists drop down  (Explorer All -> Explore All)
> 
> Since there are a lot of different lists, you may also want to consider grouping them in broader categories (like Facebook, TikTok, Twitter all under "Social Media")
> 
> I'll be submitting a pull request to fix some minor things I noticed with the github readme - is it appropriate to open an issue and link the pull request? Or just submit the pull?

Thank you very much for bringing these to our attention. Will get these fixed right away. 

More than welcome to just create a PR

**Comment by gap579137** [comment]
@avoidthehack just pushed an update to the website to fix everything you brought up. Thank you again!

**Comment by avoidthehack** [comment]
You guys move fast! I'll let you know if I have any other ideas that may be helpful! I'll be submitting a separate PR momentarily.

Nice project you got going here. :)

**Comment by blocklistproject** [comment]
We try. Especially on the remove requests. Once verified we try to remove them with in 24 hrs. Add request take a little bit longer and I am a little behind because there are really only 2 of us that do add requests.
________________________________
